src/spring/azext_spring/_validators_enterprise.py

```diff
@@ -118,17 +120,6 @@ def validate_build_service(namespace):
                 and ((namespace.registry_server is None) or (namespace.registry_username is None) or (namespace.registry_password is None)):
             raise InvalidArgumentValueError(
                 "The'--registry-server', '--registry-username' and '--registry-password' should be specified together.")
-        if (namespace.registry_server or namespace.registry_username or namespace.registry_password is not None) \
-                and namespace.disable_build_service:
-            raise InvalidArgumentValueError(
-                "Conflict detected: '--registry-server', '--registry-username' and '--registry-password' "
-                "can not be set with '--disable-build-service'.")
-        if namespace.disable_build_service:
-            namespace.disable_app_insights = True
-            if namespace.app_insights or namespace.app_insights_key:
-                raise InvalidArgumentValueError(
-                    "Conflict detected: '--app-insights' or '--app-insights-key' "
-                    "can not be set with '--disable-build-service'.")
```

Comment:
why do we delete these lines？

Reply:
Previously App Insights is disabled when build service is diabled as APM is not ready. Now App Insights can be enabled when build service is diabled as APM is ready

Reply:
OK, I think you should keep lines 121-125, which are not related to APMs.

Reply:
restored lines 121-125